antarest/study/model.py

```diff
@@ -328,12 +305,6 @@ class Study(Base):
     tags: Mapped[List[Tag]] = relationship(Tag, secondary=StudyTag.__table__, back_populates="studies")
     owner = relationship(Identity, uselist=False)
     groups = relationship(Group, secondary=StudyGroup.__table__, cascade="")
-    directory = relationship("Directory", uselist=False)
```

Comment:
Why is it removed ?

Seems it does not break any test though, so I guess it's safe to remove it